### DISCUSSION TITLE
https://obsidian-sooty.vercel.app/%E7%AC%94%E8%AE%B0/%E6%97%85%E6%B8%B8%E8%AE%B0%E5%BD%95/%E8%A5%BF%E5%8C%97%E7%8E%AF%E7%BA%BF20240401-20240406.html

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#11] [Announcements]
# https://obsidian-sooty.vercel.app/%E7%AC%94%E8%AE%B0/%E6%97%85%E6%B8%B8%E8%AE%B0%E5%BD%95/%E8%A5%BF%E5%8C%97%E7%8E%AF%E7%BA%BF20240401-20240406.html

记录回忆，知识和畅想的地方

https://obsidian-sooty.vercel.app/%E7%AC%94%E8%AE%B0/%E6%97%85%E6%B8%B8%E8%AE%B0%E5%BD%95/%E8%A5%BF%E5%8C%97%E7%8E%AF%E7%BA%BF20240401-20240406.html

**Comment by LoneRanger9528** [comment]
问题记录：
图片无法显示；文字显示无法自动换行。
